tests/unit/h/services/group_list_test.py

```diff
@@ -354,6 +356,13 @@ def sample_groups(factories, other_authority, document_uri, default_authority, u
         "private": factories.Group(creator=user),
     }
 
+    # Make `user` a member of the sample_groups["private"].
+    db_session.add(
+        GroupMembership(user_id=user.id, group_id=sample_groups["private"].id)
+    )
```

Comment:
In this case it was easier to blindly maintain the previous behavior of this shared fixture, which is that `user` is a member of `sample_group["private"]`. If you don't maintain this then several tests break. I don't know whether _all_ tests using this fixture require this user to be a member of this group, but it was easiest just to make it so.